qualang_tools/optimal_weights/configuration.py

```diff
@@ -0,0 +1,271 @@
+import numpy as np
+import matplotlib.pyplot as plt
```

Comment:
why do we have here a configuration?
If it is used by any of the classes then it needs to be internal to them and prefixed with an _

Reply:
Yes, you are right, the config, examples, should be taken to qua_libs examples